Doc/library/concurrent.futures.rst

```diff
@@ -287,7 +292,8 @@ efficient alternative is to serialize with :mod:`pickle` and then send
 the bytes over a shared :mod:`socket <socket>` or
 :func:`pipe <os.pipe>`.
 
-.. class:: InterpreterPoolExecutor(max_workers=None, thread_name_prefix='', initializer=None, initargs=(), shared=None)
+.. class:: InterpreterPoolExecutor(max_workers=None, thread_name_prefix='', \
```

Comment:
This is unrelated, but it's probably an improvement as it's now more readable. So I'm ok with it.